NomaiVR/Input/VRActionInput.cs

```diff
@@ -141,13 +141,13 @@ private bool IsOppositeHandWithSameName(VRActionInput other)
             return false;
         }
 
-        private string[] GetHoldActionText()
+        private List<string>GetHoldActionText()
         {
             if (_holdActionInput == null)
             {
-                return new string[] { };
+                return new List<string>();
             }
-            return _holdActionInput.GetText();
+            return _holdActionInput.GetText().ToList();
```

Comment:
Why this change?

Reply:
This is cause I updated Harmony and the Add to a string array at line 56 was an Extension method in Harmony.
The newer version changed namespace and name of the method to AddToArray. At the time I didn't look into Harmony's documentation. 

I can update it to use the new extension method but I notice that this is the only place where we use Harmony directly, everything else is done through OWML. I also looked at the code for Hamony's extension and it's doing effectively the same thing in memory. It's up to you to decide, by keeping this change Harmony can be removed as a dependecy.

I could also leave GetHoldActionText unchanged and replace Add with Harmony's implementation:
`return GetHoldActionText().Concat(new[] { WrapWithBrackets(result) }).ToArray();`

Reply:
Ah ok, understood. The harmony dependency thing is a leftover from when OWML didn't have some features it now has. I see how it doesn't make sense anymore. I'll leave it up to you.